src/propeller/push/instructions/polymorphic.cljc

```diff
@@ -25,11 +25,17 @@
         state
         (state/push-to-stack state stack top-item)))))
 
+;; Limits the number of items that can be duplicated onto a stack at once.
+;; We might want to extend to limit all the different that things may be
+;; placed on a stack.
+(def max-stack-items 100)
```

Comment:
We should probably created a `globals` namespace like @mcgirjau did and move this into that namespace.

Also, we chose 100 because that seemed to be what the roughly corresponding "global" in Clojush was set to. Not sure if that's a reasonable choice, though, as it seems kinda small? Maybe @thelmuth has thoughts on this?